web-ng/src/app/services/feature/feature.service.ts

```diff
@@ -93,8 +103,14 @@ export class FeatureService {
       layerId,
       new firestore.GeoPoint(lat, lng)
     );
-    return this.dataStore
-      .updateFeature(project.id, newFeature)
-      .then(() => this.selectFeature(newFeature.id));
+    await this.dataStore.updateFeature(project.id, newFeature);
+    return this.selectFeature(newFeature.id);
+  }
+
+  private updatePointInternal(project: Project, feature: Feature) {
```

Comment:
Let's make this method `async` for consistency with other new methods.

Reply:
Same as above, reverted.

Reply:
I couldn't find the comment you're referring to here.. for my own edification, why can't this be async?

Reply:
Oh, nvm, this is not async because there is no `await` in this method.

Reply:
Does this work?
```ts
  private async updatePointInternal(project: Project, feature: Feature) {
    if (project.layers.isEmpty()) {
      return;
    }
    await this.dataStore.updateFeature(project.id, feature);
  }
```